torch/csrc/jit/custom_operator.h

```diff
@@ -56,6 +57,25 @@ FunctionSchema createFunctionSchemaFromTraits(const std::string& name) {
   return {name, arguments, returns};
 }
 
+template <size_t... Is, typename... Types>
+Node* getTracedNode(
+    const FunctionSchema& schema,
+    const std::tuple<Types...>& tuple) {
+  auto symbol = Symbol::fromQualString(schema.name);
+  const auto& graph = tracer::getTracingState()->graph;
+  Node* node = graph->create(std::move(symbol), /*outputs=*/0);
+  tracer::recordSourceLocation(node);
+
+  // Hack to call addInputs for the parameter pack in a sequenced fashion.
+  // https://stackoverflow.com/questions/12030538/calling-a-function-for-each-variadic-template-argument-and-an-array
+  int _[] = {(tracer::addInputs(node, schema.arguments[Is].name.c_str(), std::get<Is>(tuple)), 0)...};
```

Comment:
This would be nicer with our previous way of adding inputs 😕

Reply:
I don't think you could expand the name and value parameter packs to be side by side, as in `(names[Is], values[Is])...` would probably not work because of the parentheses I think